block/src/main/java/nova/sample/block/NovaBlock.java

```diff
@@ -47,10 +47,10 @@
 	public final RecipeManager recipeManager;
 
 	public NovaBlock(BlockManager blockManager,
-					 ItemManager itemManager,
-					 RenderManager renderManager,
-					 NetworkManager networkManager,
-					 RecipeManager recipeManager) {
+	                 ItemManager itemManager,
+	                 RenderManager renderManager,
+	                 NetworkManager networkManager,
+	                 RecipeManager recipeManager) {
```

Comment:
The spaces being used as leading whitespace in constructors was already done like that in NovaWorldgen, I just manually copied it to the other mod classes.